lib/kvs/kvs_cursor.c

```diff
@@ -700,6 +701,29 @@ kvs_cursor_free(struct hse_kvs_cursor *cursor)
         ikvs_cursor_save(cursor_h2r(cursor));
 }
 
+static merr_t
+kvs_cursor_bh_create(struct hse_kvs_cursor *cursor)
+{
+    struct kvs_cursor_impl *cur = cursor_h2r(cursor);
+    bin_heap2_compare_fn *  cmp;
+    merr_t err;
+    int cnt = 0;
+
+    cur->kci_esrcv[cnt++] = c0_cursor_es_make(cur->kci_c0cur);
+    cur->kci_esrcv[cnt++] = lc_cursor_es_make(cur->kci_lccur);
+    cur->kci_esrcv[cnt++] = cn_cursor_es_make(cur->kci_cncur);
+
+    cmp = cur->kci_reverse ? kvs_cursor_cmp_rev : kvs_cursor_cmp;
+    if (cur->kci_bh)
+        bin_heap2_destroy(cur->kci_bh);
+
+    err = bin_heap2_create(KVS_CURSOR_SOURCES_CNT, cmp, &cur->kci_bh);
+    if (ev(err))
+        return err;
+
+    return 0;
+}
```

Comment:
I don't see any need for **cnt** in this function (I realize you copy-n-pasted this from elsewhere...)

Reply:
I suppose you could pass **cnt** into **bin_heap2_create**(), but then maybe you'd want an a**ssert(cnt == KVS_CURSOR_SOURCES_CNT)**...

Reply:
Oops. I got rid of the use of cnt but not cnt itself. Fixed it.